README.md

```diff
@@ -0,0 +1,16 @@
+# **Stock Market**
+
+## Project summary
+In this project I created a system which simulates a stock market. The stock market composed of some companies. The system enables the user to insert and remove companies from the stock market and to make different actions.
```

Comment:
system is a complicated word for this case; I would simply say that you simulated a stock market using 2 classes, there is no app or system here. 

The stock market is composed of some companies? I think you can do better here -  "The stock market class represents the entire set of companies that are found in the stock market and enables the user to conduct a variety of action the different companies present in the market" - an example.

"system enables the user to insert and remove companies from the stock market and to make different actions." - This explanation is not appealing, and does not focus on the interesting things your code can do, try - By using the api given by the classes, different mniuplations can be conducted such as changing net worth of companies, merging and more.." (merging and net worth sound alot better than inserting and removing doesnt it?)

Reply:
I really like your wording. I used it and added a sentence about 'company' class.